build/templates/storage.tpl.json

```diff
@@ -234,17 +263,332 @@
+              },
+              {
+                "columnMatch": "OSDisk",
+                "formatter": 5
+              },
+              {
+                "columnMatch": "ReadReplica",
+                "formatter": 18,
+                "formatOptions": {
+                  "thresholdsOptions": "icons",
+                  "thresholdsGrid": [
+                    {
+                      "operator": "==",
+                      "thresholdValue": "configured",
+                      "representation": "success",
+                      "text": "{0}{1}"
+                    },
+                    {
+                      "operator": "==",
+                      "thresholdValue": "not configured",
+                      "representation": "2",
+                      "text": "{0}{1}"
+                    },
+                    {
+                      "operator": "==",
+                      "thresholdValue": "not applicable",
+                      "representation": "more",
+                      "text": "{0}{1}"
+                    },
+                    {
+                      "operator": "Default",
+                      "thresholdValue": null,
+                      "representation": "unknown",
+                      "text": "{0}{1}"
+                    }
+                  ]
+                }
+              },
+              {
+                "columnMatch": "AutomaticFailover",
+                "formatter": 5
+              },
+              {
+                "columnMatch": "MultipleWriteLocations",
+                "formatter": 5
+              },
+              {
+                "columnMatch": "StorageAutogrow",
+                "formatter": 5
+              },
+              {
+                "columnMatch": "Backup",
+                "formatter": 5
+              },
+              {
+                "columnMatch": "Environment",
+                "formatter": 18,
+                "formatOptions": {
+                  "thresholdsOptions": "colors",
+                  "thresholdsGrid": [
+                    {
+                      "operator": "contains",
+                      "thresholdValue": "Prod",
+                      "representation": "redBright",
+                      "text": "{0}{1}"
+                    },
                     {
                       "operator": "contains",
-                      "thresholdValue": "v2",
+                      "thresholdValue": "prod",
+                      "representation": "redBright",
+                      "text": "{0}{1}"
+                    },
+                    {
+                      "operator": "Default",
+                      "thresholdValue": null,
+                      "text": "{0}{1}"
+                    }
+                  ]
+                }
+              },
+              {
+                "columnMatch": "SubscriptionName",
+                "formatter": 5
+              },
+              {
+                "columnMatch": "SecondaryLocation",
```

Comment:
@tsubasaxZZZ I don't see these fields (incl secondary status, access tier) in storage KQL. Am I missing something?

Reply:
Maybe these settings just stay in place. I'll remove them.